internal/internal_utils.go

```diff
@@ -275,7 +275,7 @@ func awaitWaitGroup(wg *sync.WaitGroup, timeout time.Duration) bool {
 
 func getKillSignal() <-chan os.Signal {
 	c := make(chan os.Signal, 1)
-	signal.Notify(c, os.Interrupt, syscall.SIGTERM)
+	signal.Notify(c, syscall.SIGINT, syscall.SIGTERM)
```

Comment:
I may lack some context here. But I think it's better to use `os` package if we can as it's system independent?  Also, from this page (https://golang.org/pkg/syscall/) seems like `syscall` package has been locked down.

Reply:
They are same thing so just make it unified, in os.
```
Interrupt Signal = syscall.SIGINT
```